toolset/kover-cli/src/test/kotlin/kotlinx/kover/cli/tests/SimpleTests.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2000-2023 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kotlinx.kover.cli.tests
+
+import kotlinx.kover.cli.invokeCli
+import java.io.File
+import kotlin.io.path.createTempDirectory
+import kotlin.test.Test
+
+private const val RESOURCES_PATH = "src/test/resources"
+
+class SimpleTests {
+    @Test
+    fun instrument() {
+        val classes = File("$RESOURCES_PATH/classes")
+        val dir = createTempDirectory("kover-offline-test")
+        val targetPath = dir.toFile().canonicalPath
+
+        val args = buildList {
+            add("instrument")
+            add(classes.canonicalPath)
+            add("--dest")
+            add(targetPath)
+        }
+
+        println("Off-line instrumentation with args: " + args.joinToString(" "))
+        println("Output dir file://$targetPath")
+        invokeCli(args.toTypedArray())
```

Comment:
I think these tests are incomplete:

- return code of `invokeCli` should be checked, as there are a lot of `catch`es indside
- Ideally, you should compare the resulting `targetPath` contents with some 'golden' contents. But at this stage, it is possible to check that at least some files with necessary extensions exist in the target folder.
- There are no tests for incorrect/incomplete information. E.g. basic check that `invokeCli` returns -1 when `classes.canonicalPath` is not provided or contains non-existent file

Reply:
These are temporary ones that check the basic functionality.
It is necessary to add tests that will honestly use compiled jar artifacts.

I suggest adding such tests later.